h2/src/test/org/h2/test/unit/TestFileSystem.java

```diff
@@ -53,7 +53,11 @@
     public static void main(String... a) throws Exception {
         TestBase test = TestBase.createCaller().init();
         // test.config.traceTest = true;
-        test.testFromMain();
+        FilePathEncrypt.register();
+        for (int i = 0; i < 100; i++) {
+            ((TestFileSystem)test).testConcurrent("encrypt:0007:" + test.getBaseDir() + "/fs");
+            test.println("Done pass #" + i);
+        }
```

Comment:
This temporary change shouldn't be here.

Reply:
Sorry, of course, it's unintended leftover